stewart_intro/generate_training_data.py

```diff
@@ -0,0 +1,35 @@
+from stewart_intro.utils import load_data
+
+scalar_features = ['LHF', 'SHF', 'SOLIN']
+
+
+def get_initial_shape():
+    return load_data().QT.shape
+
+
+def get_target_normalization(variable):
+    data = load_data()
+    mean_by_vertical_slice = data[variable].mean(axis=(0, 2, 3)).values
+    std_by_vertical_slice = data[variable].mean(axis=(0, 2, 3)).values
```

Comment:
you should call `.std` here. It looks like you are normalizing by the mean. 

If the shape of the array is "(time, z, y, x)" then the normalization in the paper is `var.std(axis=(0,2,3)).mean(axis=0)`. The final mean across vertical levels is needed because the variance of QT is near 0 for the upper atmosphere (cold) vertical levels, and dividing by zero is bad.

Reply:
Got it, thanks for clarifying. I'll update this.